### ISSUE TITLE
Too much encoding in core-config with D8

### ISSUE THREAD

**Issue by weitzman** [drush-ops/drush#1721]
I am seeing HTML placeholder ugliness in the choice list when running on a D8 site: `drush core-config`

**Comment by weitzman** [comment]
Assigning to @greg-1-anderson as followup from #1637

**Comment by anavarre** [comment]
If we're talking about:

```
Invalid placeholder: !default FormattableMarkup.php:240
Invalid placeholder: !type FormattableMarkup.php:240
```
Yes, seeing that with plenty of different commands (cget, ev, etc.)

**Comment by weitzman** [comment]
Fixed by 2c9471d163f39b76bdc3660563954f5ca05e3f2d
